src/components/ebay-item-tile/examples/default.marko

```diff
@@ -0,0 +1,38 @@
+class {}
+<ebay-item-tile on-action("emit", "action") href="https://ebay.com"
+        file={
+            name: "file-name.jpg",
+            type: "image/jpeg",
+            src: "https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg",
+          }
+          ...input
+>
+    <@action aria-label="Add to watchlist">
+        <ebay-heart-16-icon/>
+    </@action>
+    <@supertitle>
+      <ebay-signal status="time-sensitive">
+        Time Sensitive
+      </ebay-signal>
+    </@supertitle>
+      <@title>
+          Apple iPhone 11 Pro Max
+      </@title>
+      <@subtitle>
+          256GB Space Gray
+      </@subtitle>
+    <@description>
```

Comment:
The various subgroups in this section need to use `<p>` instead of <`div>` because Skin has built in relative line spacing DS wanted. The current spacing is incorrect.

<kbd><img width="562" alt="image" src="https://github.com/user-attachments/assets/9d820232-9fb4-4704-8827-fd43dbef1abf" /></kbd>

It may be a good idea to actually use `<@attribute>` to do the `<p>` wrapping on our end so teams can't use something that will result in spacing issues. There's a possibility for any host of attributes. That will probably also help business objects components so more brand-specific attributes continue to retain that spacing as well.

Reply:
Sounds good. Will adjust

Reply:
One comment, we should adjust skin since some examples have `<div>`
<img width="912" alt="Screenshot 2025-04-04 at 9 20 54 AM" src="https://github.com/user-attachments/assets/7664a2ce-0a9f-4cb8-935c-1da8adb3509d" />